api/types.proto

```diff
@@ -447,13 +447,25 @@ message AcceptancePolicy {
 }
 
 
+message ExternalCA {
+	enum CAType {
+		CFSSL = 0 [(gogoproto.enumvalue_customname) = "CATypeCFSSL"];
+	}
+
+	// CAType is the type of this external CA.
+	CAType type = 1;
```

Comment:
I thought it would be better to make it an enum like this. Do you think I should change it to a string?